packages/react/fast-components-react-base/src/data-grid/data-grid-cell.schema.2.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * Complies with FAST Tooling 2.0
+ */
+export default {
+    $schema: "http://json-schema.org/schema#",
+    title: "Data grid cell",
+    description: "A data grid cell's schema definition.",
+    type: "object",
+    id: "@microsoft/fast-components-react-base/data-grid-cell",
+    formPluginId: "@microsoft/fast-components-react-base/data-grid-cell",
+    properties: {
+        rowData: {
+            title: "Row data",
+            type: "object",
+        },
+        columnIndex: {
+            title: "Column index",
+            type: "number",
+        },
+        columnDefinition: {
+            title: "Column definition",
+            type: "object",
+            properties: {
+                columnDataKey: {
+                    title: "Column data key",
+                    type: "string",
+                },
+                title: {
+                    title: "Title",
+                    type: "object",
+                },
+                columnWidth: {
+                    title: "Column Width",
+                    type: "string",
+                },
+            },
+        },
+        children: {
+            title: "Children",
+            type: "children",
```

Comment:
To comply with tooling v2 you would need to use the export from `@microsoft/fast-tooling`, `linkedDataSchema`, see the `accent-button.schema.2.ts` for an example

Reply:
Only needed one schema since there are no children.